csunplugged/resources/views/sorting_network.py

```diff
@@ -49,3 +37,37 @@ def resource_image(get_request, resource):
             base_coord_x += coord_x_increment
 
     return image
+
+
+def subtitle(get_request, resource):
+    """Returns the subtitle string of the resource.
+
+    Used after the resource name in the filename, and
+    also on the resource image.
+    """
+    TEMPLATE = '{} to {}'
```

Comment:
you're a little inconsistent with naming template strings.
Please make this more descriptive than just "template"